src/python/tests/core/bot/fuzzers/libFuzzer/engine_test.py

```diff
@@ -344,6 +344,7 @@ def mock_merge(*args, **kwargs):  # pylint: disable=unused-argument
         'strategy_fork': 0,
         'strategy_mutator_plugin': 0,
         'strategy_mutator_plugin_radamsa': 0,
+        'strategy_peach_grammar_mutation': 0,
```

Comment:
The type defined in fuzzer_stats_schema is a STRING, while this is an int?

Reply:
I am not sure where this value is being set. It seems to be 0 by default. Do you know what I need to change to make this the tag?

Reply:
Most likely

https://github.com/google/clusterfuzz/blob/5e9b02204428cb1a81993d512844966d2bee0a80/src/python/bot/fuzzers/libFuzzer/stats.py#L205